suseRegisterInfo/suseRegisterInfo.spec

```diff
@@ -37,7 +37,7 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildArch:      noarch
 %endif
 Requires:       %{pythonX}-%{name} = %{version}-%{release}
-Requires:       perl
+Requires:       perl-base
```

Comment:
perl-base exists for SUSE only. RedHat does not have it. So we need to add some macros.
For SUSE, use perl-base. All other still just "perl"

Reply:
Ah, I was under the impression that this tool is only used on SUSE systems (because of its name). Will make the change distribution dependent.